Presentation.Web/Controllers/API/V2/External/ItSystems/ItSystemDBSV2Controller.cs

```diff
@@ -28,10 +28,10 @@ public ItSystemDBSV2Controller(IItSystemWriteService writeService, IDBSWriteMode
     [SwaggerResponse(HttpStatusCode.Forbidden)]
     [SwaggerResponse(HttpStatusCode.NotFound)]
     [SwaggerResponse(HttpStatusCode.Unauthorized)]
-    public IHttpActionResult PatchDbsProperties([NonEmptyGuid][FromUri] Guid systemUuid, [FromBody] UpdateDBSPropertiesRequestDTO request)
+    public IHttpActionResult PatchDbsProperties([NonEmptyGuid][FromUri] Guid systemUuid, [FromBody] LegalPropertyUpdateRequestDTO request)
```

Comment:
Tiny thing, for the name of the DTO and the mapper - maybe they should be called LegalPropertIES not PropertY since they are multiple, same as the writeService method call